PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -445,7 +446,15 @@ extension DefaultControlsViewController {
                 guard case .player(let player) = props else { return true }
                 guard case .playable(let props) = player.item else { return true }
                 
-                return props.isLive == false
+                return props.live.isHidden
+            }()
+            
+            liveDotColor = {
+                guard
+                    case .player(let player) = props,
```

Comment:
Better to use prism style `props.player?.foo.boo`

Reply:
👍

Reply:
@BogdanBilonog Done.